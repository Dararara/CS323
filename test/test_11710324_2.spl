int test_1_m01(int a int b)
{
  int code, new;
  code 1;
  new = code + 1;
  return hello('a, new);
}