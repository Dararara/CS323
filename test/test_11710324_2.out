Error type B at Line 1: Missing comma ','
Error type B at Line 4: Missing semicolon ';'
Error type B at Line 6: Missing closing '
